docs/integrations/saas-cloud/confluent-cloud.md

```diff
@@ -0,0 +1,119 @@
+
+Confluent Cloud - Compute Pool dashboard provides details on the status and utilization of compute resources in your Confluent Cloud environment. Use this dashboard to:
+
+* Monitor the average Flink statement status to identify potential issues with stream processing jobs.
+* Track CPU utilization against the set limit to ensure optimal resource allocation and prevent performance bottlenecks.
+* Analyze trends in CPU minutes consumed over time to optimize cost and capacity planning for your Confluent Cloud deployment.
+* Compare current CPU usage across different compute pools to balance workloads and improve overall system efficiency.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ConfluentCloud/Confluent-Cloud-Compute-Pool.png')} alt="Confluent Cloud - Compute Pool" style={{border: '1px solid gray'}} width="800" />
+
+### Confluent Cloud - Connectors
+
+Confluent Cloud - Connectors dashboard provides details on the status, performance, and health of Confluent Cloud connectors.
+
+Use this dashboard to:
+* Monitor the status of connectors and their tasks to quickly identify any that are failed, paused, or provisioning.
+* Track the volume of sent and received records across different connectors to ensure data flow is as expected.
+* Analyze the average sent and received bytes to detect any unusual spikes or drops in data transfer.
+* Identify potential bottlenecks by correlating connector status with sent/received records and bytes.
+* Monitor dead letter queue records to catch any messages that failed to be processed correctly.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ConfluentCloud/Confluent-Cloud-Connectors.png')} alt="Confluent Cloud - Connectors" style={{border: '1px solid gray'}} width="800" />
+
+### Confluent Cloud - Kafka Cluster
+
+Confluent Cloud - Kafka Cluster dashboard provides details on key performance metrics and operational statistics for your Kafka cluster in Confluent Cloud.
+
+Use this dashboard to:
+* Monitor producer latency and cluster load to ensure optimal performance of your Kafka cluster.
+* Track active connections, request bytes, and response bytes to identify potential bottlenecks or unusual traffic patterns.
+* Analyze request count trends over time to understand usage patterns and plan for capacity needs.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ConfluentCloud/Confluent-Cloud-Kafka-Cluster.png')} alt="Confluent Cloud - Kafka Cluster" style={{border: '1px solid gray'}} width="800" />
+
+### Confluent Cloud - Kafka Cluster Links
+
+Confluent Cloud – Kafka Cluster Link dashboard provides key insights into cluster link performance and health. It tracks link task counts, mirror topic states, offset lags, data volume, and response metrics—helping monitor cross-cluster replication efficiency, detect mirror errors, and ensure seamless data delivery across environments.
```

Comment:
apply the same change as https://github.com/SumoLogic/sumologic-documentation/pull/5495/files#r2172504296